src/intfuncs.c

```diff
@@ -620,6 +620,293 @@ Obj FuncINT_STRING ( Obj self, Obj string )
 
 /****************************************************************************
 **
+*F SmallInt Bitmask operations
+*
+*  Since all our functions have special handlers and no BODY, NLOC is unused,
+*  so we repurpose it.
+*
+*/
+
+#ifdef SYS_IS_64_BIT
+#define LENSTART_WIDTH 8
+#define WIDTH_WIDTH 6
+#define MAX_FIELDS 10
+#else
+#define LENSTART_WIDTH 8
+#define WIDTH_WIDTH 5
+#define MAX_FIELDS 6
```

Comment:
We should have plenty of fields to store things in: ENVI_FUNC, NLOC_FUNC, BODY_FUNC, FEXS_FUNC